dandi/core/digests/dandietag.py

```diff
@@ -0,0 +1,169 @@
+# Derived from <https://github.com/girder/django-s3-file-field/blob/master/
+# s3_file_field/_multipart.py>, copyright Kitware, Inc. <kitware@kitware.com>
+# under the Apache 2.0 license
```

Comment:
Please also add to `## 3rd party components included` of README.md where we have similar mentions ATM (we might want to move them to LICENSE later on)